### PR TITLE
Data Analytics: Fivetran & Snowflake & Tableau

### PR DESCRIPTION
**Pull request by swiknaba**
#### Summary

We need a proper set-up for data exploration across multiple sources and projects. For this, we agreed on using the following stack:

* Fivetran for ETL
* Snowflake as Data Warehouse
* Tableau for viewing the data, generating dashboards, etc.

We are still exploring DBT for creating and maintaining views on the data.

#### Motivation

We want to make data driven decisions.

#### Test plan

This will be rolled out and tested on our internal dummy application.

#### Rollout/monitoring/revert plan

-/-

### CODE REVIEW

File: analytics/secrets.tf [hunk 1]
```diff
@@ -0,0 +1,25 @@
+module "kms-key" {
+  source = "../kms-key"
+
+  # Required
+  environment  = local.environment
+  project      = local.project
+  alias        = "analytics"
```
Comment (samkahchiin) [inline analytics/secrets.tf:7]: alias have to be unique, perhaps make it project/env specific or make it a variables (with analytics as the default value)?

File: snowflake/cloud/README.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+|--------------|---------|-------|--------|-------|---------|----------|----------|----------|
+| Credits/hour | 1       | 2     | 4      | 8     | 16      | 32       | 64       | 128      |
+
+
+
+## Prerequisites
+
+Sign up for a Snowflake account. Use a generic email address like `snowflake@my-company.com` to decouple the root access from a specific employee's account.
+
+During the "initial set up" a bot account is created with less-than-root-level access for managing the accout through terraform.
```
Comment (samkahchiin) [inline snowflake/cloud/README.md:56]:
```suggestion
During the "initial set up" a bot account is created with less-than-root-level access for managing the account through terraform.
```

File: snowflake/cloud/versions.tf [hunk 3]
```diff
@@ -0,0 +1,49 @@
+terraform {
+  required_providers {
+    snowflake = {
+      source                = "Snowflake-Labs/snowflake"
+      version               = "~> 0.35"
```
Comment (samkahchiin) [inline snowflake/cloud/versions.tf:5]: Latest version? 

```suggestion
      version               = "~> 0.42"
```

File: analytics/secrets.tf (hunk 1 above)
Comment (swiknaba) [reply analytics/secrets.tf:7]: yes! good point :)
the KMS module takes care of this for us
> name          = replace("alias/${var.project}/${var.environment}/${var.alias}", "/[^a-zA-Z0-9:///_-]+/", "-")

**Pull request merged**